presto-spark-base/src/test/java/com/facebook/presto/spark/execution/http/TestPrestoSparkHttpWorkerClient.java

```diff
@@ -95,6 +97,9 @@
             .port(8080)
             .build();
     private static final Duration NO_DURATION = new Duration(0, TimeUnit.MILLISECONDS);
+    private static final JsonCodec<TaskInfo> taskInfoCodec = JsonCodec.jsonCodec(TaskInfo.class);
+    private static final JsonCodec<PlanFragment> planFragmentCodec = JsonCodec.jsonCodec(PlanFragment.class);
+    private static final JsonCodec<TaskUpdateRequest> taskUpdateRequestCodec = JsonCodec.jsonCodec(TaskUpdateRequest.class);
```

Comment:
We use upper cases + underscores for static variables. For example `TASK_INFO_JSON_CODEC`